### PR TITLE
Fix kraken products decimal precision in update-products.sh

### PR DESCRIPTION
**Pull request by MT00x**
Fix automatic products generation. (update-products.sh)
Fix commit  #757  that overwrote minimum increment for Kraken products
Restored changes in "Fix kraken products for krakens update on decimal precision #616"

### CODE REVIEW

**Review by storm1er — APPROVED** [review]

File: extensions/exchanges/kraken/exchange.js [hunk 1]
```diff
@@ -1,352 +1,362 @@
 var KrakenClient = require('kraken-api'),
```
Comment (DeviaVir) [inline extensions/exchanges/kraken/exchange.js]: Did you mean to completely redo the indenting of this file?

Comment (MT00x) [reply extensions/exchanges/kraken/exchange.js]: That's my fault on last commit on exchange.js on my fork, but this commit is not included in this pull request. Only update-products.sh and products.json should be merged.

Anyway, if you are interested in my fix for kraken connection error I'll fix the indenting asap

Comment (DeviaVir) [reply extensions/exchanges/kraken/exchange.js]: Sure am, so please do :)

Comment (MT00x) [reply extensions/exchanges/kraken/exchange.js]: ok :) this evening. I'll ping you when done

Comment (MT00x) [reply extensions/exchanges/kraken/exchange.js]: Fixed indenting and
I tried to fix Issue #800 , please look at line 349 in exchange.js

```
order.price = n(orderData.price).format('0.00000000')
```

Tell me what you think about it  :)

File: extensions/exchanges/kraken/exchange.js [hunk 2]
```diff
@@ -1,352 +1,363 @@
...
+  var c = get('conf')
+  var s = {
+    options: minimist(process.argv)
+  }
+  var so = s.options
+
+ var public_client, authed_client
+ // var recoverableErrors = new RegExp(/(ESOCKETTIMEDOUT|ETIMEDOUT|ECONNRESET|ECONNREFUSED|ENOTFOUND|API:Invalid nonce|API:Rate limit exceeded|between Cloudflare and the origin web server)/)
+ var recoverableErrors = new RegExp(/(ESOCKETTIMEDOUT|ETIMEDOUT|ECONNRESET|ECONNREFUSED|ENOTFOUND|API:Invalid nonce|between Cloudflare and the origin web server|The web server reported a gateway time\-out|The web server reported a bad gateway|525\: SSL handshake failed)/)
+ var silencedRecoverableErrors = new RegExp(/(ESOCKETTIMEDOUT|ETIMEDOUT)/)
```
Comment (DeviaVir) [inline extensions/exchanges/kraken/exchange.js]: still wrong indented here

**Review by DeviaVir — APPROVED** [review]
:+1:

File: extensions/exchanges/kraken/exchange.js (hunk 2 above)
Comment (MT00x) [reply extensions/exchanges/kraken/exchange.js]: Sorry for that :( fixed

**Pull request merged**